src/main/java/com/googlecode/lanterna/gui2/AbstractListBox.java

```diff
@@ -535,7 +637,8 @@ public String getLabel(T listBox, int index, V item) {
          */
         public void drawItem(TextGUIGraphics graphics, T listBox, int index, V item, boolean selected, boolean focused) {
             ThemeDefinition themeDefinition = listBox.getTheme().getDefinition(AbstractListBox.class);
-            if(selected && focused) {
+            //if(selected && focused) {
+            if (selected) {
```

Comment:
Was this change intentional?

Reply:
Yes, it is so that the ListBox item under the mouse is highlighted, it would be the one 'activated' when clicked.  However due to another change in this PR it remains dormant still unfortunately.
Since now events are routed to only the active window.